### PR TITLE
Feature/edit sheet name 98674692

### PR DESCRIPTION
**Pull request by sidke**
inactive:
image

renaming a sheet:
image

adding a new sheet:
image
image

### CODE REVIEW

File: assets/js/utils/optimizeResize.coffee [hunk 1]
```diff
@@ -0,0 +1,33 @@
+@boardroom.utils.optimizedResize = (->
```
Comment (sueyeh) [inline assets/js/utils/optimizeResize.coffee]: What is this util getting used for? Also the filename should match the util name (optimize vs optimized)

File: assets/js/views/edit_name.coffee [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    @listenTo @model, 'change:name', @updateName
+
+  render: ->
+    @
+
+  remove: ->
+    super()
+
+  initializeLock: =>
+    @editLock = @createEditLock @el
```
Comment (sueyeh) [inline assets/js/views/edit_name.coffee]: What is the purpose of these locks?

**Comment by sueyeh** [comment]
lgtm. rebasing might break some specs, since we added role to collaborator.

File: assets/js/utils/optimizeResize.coffee (hunk 1 above)
Comment (sidke) [inline assets/js/utils/optimizeResize.coffee]: doesn't look like this is getting used at all, removing it for now

File: assets/js/views/edit_name.coffee (hunk 2 above)
Comment (sidke) [inline assets/js/views/edit_name.coffee]: they're not currently showing anything it seems for board and sheet names, but they are most visible when editing a card. the edit locks are responsible for showing "Bert Bacharach is typing..." for editing group names or card content, as well as "Bert Bacharach" for moving of groups

**Pull request merged**